modelling/src/main/java/org/axonframework/modelling/command/legacyjpa/GenericJpaRepository.java

```diff
@@ -274,6 +277,12 @@ public Builder<T> spanFactory(SpanFactory spanFactory) {
             return this;
         }
 
+        @Override
```

Comment:
Does the `@Deprecated` method bubble up through an override? If not, I think we should also deprecate the override of the `spanFactory(SpanFactory)` method.

Reply:
Deprecated has been put on all overridden methods